src/azure-cli/azure/cli/command_modules/resource/tests/latest/recordings/test_tag_update_by_patch.yaml

```diff
@@ -8767,4 +8767,4 @@ interactions:
     status:
       code: 200
       message: OK
-version: 1
\ No newline at end of file
+version: 1
```

Comment:
Why do you modify this file?

Reply:
I think this was just VSCode removing the newline at the end of the file?

Reply:
Maybe